cyctl/internal/kubeconfig/kubeconfig.go

```diff
@@ -21,12 +25,18 @@ func GetKubeConfig() func(cmd *cobra.Command, args []string) {
 			kubeconfig = ""
 		}
 
-		config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+		var err error
+		Config, err = clientcmd.BuildConfigFromFlags("", kubeconfig)
 		if err != nil {
 			panic(err.Error())
 		}
 
-		Moduleset, err = client.NewForConfig(config)
+		Moduleset, err = client.NewForConfig(Config)
+		if err != nil {
+			panic(err.Error())
+		}
+
+		Clientset, err = client.NewForConfig(Config)
```

Comment:
You need to use `kubernetes` package. The `client` package does not implement operations over other k8s resources
`Clientset, err = kubernetes.NewForConfig(Config)`

Reply:
Sorry about that. I was running the cluster on a different machine and I forgot to change that